game/src/contrib/utils/components/skill/DamageProjectile.java

```diff
@@ -81,8 +80,15 @@ public void accept(Entity entity) {
                                         () -> new MissingComponentException("PositionComponent"));
         new PositionComponent(projectile, epc.getPosition());
 
-        Animation animation = AnimationBuilder.buildAnimation(pathToTexturesOfProjectile);
-        new DrawComponent(projectile, animation);
+        try {
+            new DrawComponent(projectile, pathToTexturesOfProjectile);
```

Comment:
hier könntest du ja auch prima eine der neuen Builder-Methoden `DrawComponent#of` benutzen :)

Reply:
Jetzt hast du aber was zusammengeworfen. 
`DrawComponent` != `Animation`